app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchGreetings } from '../redux/greetings/greetingsSlice';
+
+const Greeting = () => {
+	const dispatch = useDispatch();
```

Comment:
- [ ] Great job so far. However, there's spelling error with this file name. This file name was named `grettingsSlice.js` but was referenced here as `greetingsSlice.js`. Kindly correct that.

### Spelling mistake
![greetings-error](https://github.com/akezeth/hello-rails-react/assets/26597432/61e695ba-af59-4071-a2fa-dc81785dc4c3)